### PR TITLE
Create Start-Up.md

### PR DESCRIPTION
**Pull request by TangentFoxy**
Notes:

- Will want to put an image in showing basic cockpit layout.
- Has commented-out references to other docs I think should be created.

### CODE REVIEW

File: Start-Up.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+On the right panel:
+
+1. Flip "Main Battery" switch to "On"
+2. Flip "Brake Lock" switch to "On" (this is so you don't accidentally roll while starting up)
+3. Flip "APU" switch to "On" (it is a covered switch, so you need to lift the cover first)
+4. Wait for the RPM gauge under the "APU" switch to spool up
+5. Flip the covered switch for one of the engines to "On" (you can try both at once, but you are likely to have start failures)
+6. Wait for the engine to spool up (RPM gauges for the engines are on the front panel on the right side)
+7. Flip the covered switch for the other engine to "On" and wait for it to spool up
```
Comment (dhleong) [inline Start-Up.md:15]: Suggestion: "Repeat steps 5. and 6. for the other engine"

Also, perhaps add a step (or just a note) to close the covered switch after it's spooled.

File: Start-Up.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+8. Flip "APU" switch to "Off" (you don't need it once the engines are running)
+
+Front panel:
```
Comment (dhleong) [inline Start-Up.md:18]: Perhaps we should have two sections here, instead: the one above for *Starting the Engines* and a section here for *Getting the Jet Mission-Ready* with a note that the steps aren't required and may be mission-specific or personal preference, that can include turning on MFDs, turning on the HMCS, etc.

File: Start-Up.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Start-Up Procedure
```
Comment (dhleong) [inline Start-Up.md:1]: Maybe start at `##` header level, in case we ever want to merge topics together (see my review summary comment)

**Review by dhleong — COMMENTED** [review]
Thanks for getting a start on this project! Just put in my two cents on organization for this document. I think we might eventually want a combined "Quick Start Checklists" (or something) document that can be made into a PDF, but for now it might be more sane to keep topics separated as you're doing. It might also be possible to merge the separate topic files when building the PDF, so starting at the second level header might make sense for these topic files.

**Comment by BahamutoD** [comment]
I merged it but moved it into the AV-42C folder.

File: Start-Up.md (hunk 1 above)
Comment (TangentFoxy) [reply Start-Up.md:15]: > 7. Repeat steps 5 & 6 for the other engine, and close the covers.

What do you think of this for a replacement for 7 ?

File: Start-Up.md (hunk 2 above)
Comment (TangentFoxy) [reply Start-Up.md:18]: I agree. Have you looked at the comments in the file itself about referencing other files? It should probably be rolled into those. Maybe a "General Mission Readiness" and then specifics for different kinds of take-offs.

We don't want to split things into *too* many files in my opinion though, lots of unnecessary clicking between pages.

File: Start-Up.md (hunk 1 above)
Comment (dhleong) [reply Start-Up.md:15]: Sounds good to me 👍

File: Start-Up.md (hunk 2 above)
Comment (dhleong) [reply Start-Up.md:18]: > We don't want to split things into too many files in my opinion though, lots of unnecessary clicking between pages.
 Write

Oh agreed. I didn't mean separate pages, but different headers like:

```markdown
## Starting the Engines

1. Steps

## Getting the Jet Mission-Ready

2. Other steps
```

etc.

**Pull request merged**